src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -193,6 +197,11 @@ public void verifyInstructorEditFormDisabled(int instrNum) {
         assertTrue(isNotEditable);
     }
 
+    public boolean verifyShowNewInstructorIsDisplayed() {
```

Comment:
i dont really like the name of this function... you want to verify the form is shown... so you can consider `verifyAddNewInstructorFormDisplayed` or something like that, depending on how you name the action of cancellng adding a new instructor